paper.md

```diff
@@ -76,19 +76,21 @@ Finally, as both data management plan and data protection impact assessment are
 
 ## Creating a DPIA Knowledge Model
 
-*TODO: (Pinar), Vilém, Nene, and Paulette - what was the process, how you worked, what is the result*
+The DPIA knowledge model is mainly inspired by the several existing documents, namely the [CNIL DPIA document template](https://www.cnil.fr/sites/default/files/atoms/files/cnil-pia-2-en-templates.pdf), the DPIA document template used by the Luxembourg Centre for Systems Biomedicine and [ICO guidance](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/data-protection-impact-assessments-dpias/how-do-we-do-a-dpia/#how)) on data protection.
```

Comment:
- ...'is mainly inspired by several...': remove the before several
- and the ICO...: add the before ICO